src/main/java/sirius/biz/tycho/dashboards/EntitiesKeyMetricProvider.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tycho.dashboards;
+
+import sirius.db.mixing.Entity;
+import sirius.db.mixing.Mixing;
+
+import java.util.function.Consumer;
+
+/**
+ * Provides a base implementation which provides key metrics for an entity class.
```

Comment:
Could maybe describe what a key metric actually is. Or link to a place where it is described.